tailcfg/tailcfg.go

```diff
@@ -188,7 +188,7 @@ type Node struct {
 	Sharer UserID `json:",omitempty"`
 
 	Key          key.NodePublic
-	KeyExpiry    time.Time
+	KeyExpiry    time.Time                  // zero if this node does not expire (e.g. tagged nodes)
```

Comment:
I'd remove the "tagged nodes" bit. That's a bit of a distraction.

And maybe say "zero value" to help people avoid thinking 0 unix epoch time?  "zero value means no expiry" ?